cms/static/sass/views/_course-create.scss

```diff
@@ -68,6 +68,10 @@
       width: 100%;
     }
 
+    .new-course-org{
```

Comment:
is `new-library-org` missing here?

Reply:
In fact, I need to verify if this change is applied correctly but I don't know how to compile those SCSS files. Do you know who can give me a hand with the styles?

Reply:
If you're using tutor you could try [this](https://docs.tutor.overhang.io/dev.html#setting-up-a-development-environment-for-edx-platform) from the docs and then inspect the browser. You could do a before and after and post it under this comment

Reply:
Thanks, you were right:
![image](https://user-images.githubusercontent.com/33465240/197867586-ff21aea1-89f1-48dd-956c-bf3a88c8a3e0.png)


![image](https://user-images.githubusercontent.com/33465240/197868673-a69b8ba5-e2c4-4d76-89eb-fc7bac1976fa.png)